scripts/deployment/ship.sh

```
@@ -27,14 +27,15 @@ cd "$PROJECT_ROOT"
 # Load libraries
 source "$SCRIPT_DIR/lib/common.sh"
 source "$SCRIPT_DIR/lib/lock.sh"
+source "$SCRIPT_DIR/lib/deploy-contract.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./lib/deploy-contract.sh was not specified as input (see shellcheck -x).

(SC1091)